frame/system/src/lib.rs

```diff
@@ -293,15 +341,9 @@ pub mod pallet {
 		type Lookup: StaticLookup<Target = Self::AccountId>;
 
 		/// The block header.
+		#[pallet::no_default]
```

Comment:
Actually, I think this can and should have a default: there is a default header defined in `mocking` module in this crate.

Reply:
hmm I see the default header in the `mocking` file but I'm not actually sure how to use it here. In the mocking file it is defined as:

```rust
generic::Header<<T as crate::Config>::BlockNumber, sp_runtime::traits::BlakeTwo256>
```

Reply:
I provided one, in case you are interested to see how.